services/ui/package.json

```diff
@@ -7,6 +7,5 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
```

Comment:
Expecting `  "packageManager": "yarn@4.12.0",` in the corresponding package.json